terraform/deployments/github/repos.yml

```diff
@@ -298,6 +298,16 @@ repos:
         - Format
         - Type checks
 
+  govuk-chat-gradio-prototype:
+    homepage_url: "https://docs.publishing.service.gov.uk/apps/govuk-chat-gradio-prototype.html"
```

Comment:
This 404s you want: https://docs.publishing.service.gov.uk/repos/govuk-chat-gradio-prototype.html as it's not an app